src/FNA3D_Driver_Vulkan.c

```diff
@@ -4869,22 +4867,27 @@ static uint8_t VULKAN_INTERNAL_FindAvailableTextureMemory(
 
 	/* No suitable free regions exist, allocate a new memory region */
 
-	allocationResult = 2;
-	while (allocationResult == 2)
+	if (requiredSize > allocator->nextAllocationSize)
 	{
-		allocationResult = VULKAN_INTERNAL_AllocateTextureMemory(
-			renderer,
-			memoryTypeIndex,
-			image,
-			requiredSize,
-			dedicatedRequirements.prefersDedicatedAllocation,
-			&textureBuffer
-		);
+		/* allocate a page of required size aligned to ALLOCATION_SIZE increments */ allocator->nextAllocationSize
+		=
```

Comment:
Looks like a newline went rogue...

```suggestion
		/* allocate a page of required size aligned to ALLOCATION_SIZE increments */
		allocator->nextAllocationSize =
```